base/strings/basic.jl

```diff
@@ -68,6 +68,9 @@ julia> "Hello " * "world"
 ```
 """
 (*)(s1::AbstractString, ss::AbstractString...) = string(s1, ss...)
+(*)(c::Char, s::AbstractString) = string(c, s)
+(*)(s::AbstractString, c::Char) = string(s, c)
+(*)(c1::Char, c2::Char) = string(c1, c2)
```

Comment:
We might subsume all these with 

```julia
(*)(s::AbstractString, ss::Union{Char,AbstractString}...) = string(s, ss...)
(*)(c::Char, ss::Union{Char,AbstractString}...) = string(c, ss...)
```

and take advantage of the `string(::Union{Char,AbstractString}...)` method.